vsintegration/src/SupportedRuntimes/SupportedRuntimes.xml

```diff
@@ -11,6 +11,7 @@
         <Assembly Version="4.3.1.0" Description="F# 3.1 (FSharp.Core, 4.3.1.0)"/>
         <Assembly Version="4.4.0.0" Description="F# 4.0 (FSharp.Core, 4.4.0.0)"/>
         <Assembly Version="4.4.1.0" Description="F# 4.1 (FSharp.Core, 4.4.1.0)"/>
+        <Assembly Version="4.4.3.0" Description="F# 4.3 (FSharp.Core, 4.4.3.0)"/>
```

Comment:
This addition is wrong.  F# 4.3 doesn't exist. You can use ``F# 4.1 (FSharp.Core, 4.4.3.0)``